src/components/chat/chat.js

```diff
@@ -150,11 +115,15 @@ class Chat extends Component {
     this.props.socket.emit('hang up', {roomId: this.state.roomId});
   }
 
+  updateKarma = (karma) => {
+    this.props.socket.emit('update karma', {tutor: this.state.targetTutor, karma: karma })
```

Comment:
Since we're sending only the user_id and not the hole user, I would suggest to change tutor for tutor_id (just so the code is cleaner)